src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html

```diff
@@ -1,7 +1,9 @@
 <div class="panel panel-default panel-body"
-    ${disabled} id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    data-length-text-id="responseLength-${questionIndex}-${responseIndex}" data-recommended-text="${recommendedLength}">
+     data-is-session-closed="${isSessionClosed}"
+     id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+     name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+     data-length-text-id="responseLength-${questionIndex}-${responseIndex}"
+     data-recommended-text="${recommendedLength}">
```

Comment:
Yes, we try not to include unrelated changes in a PR. If you notice minor refactorings but doing them will make the PR diff bigger, either do it as a separate PR or create a `firstTimer` issue for it.